### ISSUE TITLE
Difference between inline and block extensions?

### ISSUE THREAD

**Issue by bohwaz** [Amaury/SkrivMarkup#23]
I find that the two different syntaxes for inline and block extensions is too complicated.

Why can't we have the same syntax for both? Example:

```
<<lipsum|2>>

<<translate|french|english
manger
>>
```

I also find it rather disturbing that "inline" extensions (like lipsum) can generate block content. eg. lipsum generates paragraphs. If an extension is "inline" shouldn't it only generate inline content?

If the syntax difference between inline and block extensions was to be removed it would be easier to understand, because either it's inline and the extension tag doesn't have any content except for parameters, either it's block and it has content over multiple lines.

I also just noticed that in the repository the lipsum extension is a block one, but is shown as an inline one on the syntax documentation page...

**Comment by Amaury** [comment]
You're right, the documentation is confusing. In fact, the real specification of lipsum extension is on its [proposal page](http://markup.skriv.org/proposal/show/3), where it's defined as a block one. I should change the documentation to reflect that.
And yes, inline extensions must generate inline HTML tags only.
